joomla/joomla-cms
# [4.0] Fix Codemirror as a custom elements

Pull Request for Issue #20679 

### Summary of Changes

As @okonomiyaki3000 Suggested to remove the fullscreen message. I removed it.

### Testing Instructions
1. Set default editor to codemirror in global configuration.

2. Open com_content and start creating an article. See, that there are no errors in web console.

3. Open the file `administrator/components/com_content/forms/article.xml` and  the text

```
<field
name="articletext2"
type="editor"
label="COM_CONTENT_FIELD_ARTICLETEXT_LABEL"
filter="JComponentHelper::filterText"
buttons="true"
/>
```
after the field with the name articletext

4 . Open file
<code>/administrator/components/com_content/tmpl/article/edit.php</code>

and add the text

`<?php echo $this->form->getInput('articletext2'); ?>`

after the text

`<?php echo $this->form->getInput('articletext'); ?>`

### Expected result

No error in console.

Fix codemirror as a customelements .
Credit goes to @dgrammatiko joomla/joomla-cms#20679 (comment)


### Actual result
`/media/vendor/codemirror/mode/text/html/text/html.js:1 Failed to load resource: the server responded with a status of 404 (Not Found)`
